packages/flutter/lib/src/widgets/text.dart

```diff
@@ -781,6 +787,12 @@ class Text extends StatelessWidget {
         child: ExcludeSemantics(excluding: semanticsLabel != null, child: result),
       );
     }
+    if (typographySettings != null && typographySettings.paragraphSpacing != null) {
+      result = Padding(
```

Comment:
I'm curious to hear people's thoughts about implementing `paragraphSpacing` like this. On the web the extension and bookmarklet i've been using implement it by using `margin-bottom` on the `p` element.

Reply:
I "think" Text is refering to `div` and TextSpan is refering to `p`. I think gpay has internal code that turns html into TextSpans

Reply:
Interesting, if they're converting for example `<p> ` elements into `TextSpan`s with the expectation that those `<p>` elements have `paragraphSpacing` applied to them, then that will likely not work. They would have to instead turn `<p>` elements into `Text` widgets for that expectation to be true.